discord/commands/options.py

```diff
@@ -196,6 +197,8 @@ def __init__(
         if input_type_is_class and issubclass(input_type, (Enum, DiscordEnum)):
             if description is None:
                 description = inspect.getdoc(input_type)
```

Comment:
What is the point of using `getdoc`? I think `cleandoc` is a better choice here. `getdoc` calls `cleandoc` but it also inherits docstrings from parents if the class does not have one. There is no reason that someone would want the description of their command to be the docstring for `Enum` or `DiscordEnum`.
```suggestion
                description = inspect.cleandoc(input_type.__doc__)
```